src/screens/Likes.jsx

```diff
@@ -29,6 +33,7 @@ const Likes = () => {
   return (
     <div className={styles.content}>
       <Swiper items={likes} callback={removeLike}></Swiper>
+      {!!match && <Match match={match.lab} />}
```

Comment:
`!!match` will always be true as long you have a match, the problem is that `match.lab` will return an array when you have multiple matches, and then your code will not meet the requirement anymore.